### DISCUSSION TITLE
ssl.SSLWantReadError: The operation did not complete (read) while using REST-API

### DISCUSSION THREAD

**Discussion by danielkretschmer** [encode/httpx#2615] [Potential Issue]
Hello,

I'm trying to retrieve details about Zoom accounts via their API, but ran into a strange problem.
There are a bunch of Zoom accounts I can retrieve information just fine, but there are some that throw the ssl.SSLWantReadError. It shouldn't be a problem with the endpoint as I can retrieve details about the same account via other http-clients.

```
File "C:\Program Files\WindowsApps\PythonSoftwareFoundation.Python.3.11_3.11.752.0_x64__qbz5n2kfra8p0\Lib\ssl.py", line 921, in read
    v = self._sslobj.read(len)
        ^^^^^^^^^^^^^^^^^^^^^^
ssl.SSLWantReadError: The operation did not complete (read) (_ssl.c:2546)

```

Thanks in advance for any help or ideas.

**Comment by ydennisy** [comment]
+1 for this issue - I reduced it's occurrence by setting `verify=False`
